scripts.d/ta/135_checkdns.sh

```diff
@@ -4,20 +4,20 @@ DESCRIPTION="Check DNS configuration..."
 SCRIPT_TYPE="parallel"
 
 # Checking OS internal DNS servers
-current_dns_server=`cat /etc/resolv.conf | grep -i nameserver | awk {'print $2'}`
-if [ ! -z $current_dns_server ]; then
-	echo "Found DNS server with IP address of: $current_dns_server"
+CURRENT_DNS_SERVER=$(grep -w -m 1 nameserver /etc/resolv.conf | awk {'print $2'})
+if [[ ! -z $CURRENT_DNS_SERVER ]]; then
+	echo "Found DNS server with IP address of: $CURRENT_DNS_SERVER"
 	which nslookup &> /dev/null
 	if [ $? -eq 1 ]; then
```

Comment:
Not a big deal, but you could try running nslookup -version, instead of relying on which, and condense lines 10 and 11: `if ! nslookup -version &> /dev/null; then`